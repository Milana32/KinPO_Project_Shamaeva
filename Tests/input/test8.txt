 2 _- 3 + 4 * 2 7 + 9 * <= 