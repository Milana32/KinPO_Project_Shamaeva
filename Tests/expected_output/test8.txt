((_- 2) + 3) * 4 <= (2 + 7) * 9
! ((2 + 7) * 9 < ((_- 2) + 3) * 4)